projects/angular/components/ui-suggest/src/ui-suggest.component.ts

```diff
@@ -1294,6 +1297,14 @@ export class UiSuggestComponent extends UiSuggestMatFormFieldDirective
             : this.defaultValue;
     }
 
+    onOptionsDropdownTabPressed(event: Event) {
+        if (this.isOpen && !this.expandInline) {
+            event.preventDefault();
+            this._focusNextElement();
```

Comment:
After looking into the [source code](https://github.com/angular/components/blob/d10c189f2562733fa711012a2b345abfeab573f4/src/material/select/select.ts#L1008-L1009) for the `mat-select` component I figured out that if you simply focus here the `.display` item it will work as expected. So just do this instead:
```suggestion
            this.suggestContainer.nativeElement.querySelector('.display').focus();
```
but of course use `@ViewChild` instead of querying the element

Reply:
Changed it